labs/gnmic-prometheus-grafana/group_vars/OM_PES.yaml

```diff
@@ -0,0 +1,12 @@
+type: l3leaf
+custom_structured_configuration_list_merge: append
+custom_structured_configuration_vlan_interfaces:
```

Comment:
why do we need this as custom_structured? I'd prefer a better way if possible, to avoid people replicating that.